contrib/windeploy/win-codesign.cert

```diff
@@ -1,112 +1,131 @@
+T9DnMWBHVNIabGqgQSGTrQWo43MOfsPynhbz2Hyxf5XWKZpRvr3dMapandPfYgoZ
+8iDL2OR3sYztgJrbG6VZ9DoTXFm1g0Rf97Aaen1l4c+w3DC+IkwFkvjFV3jS49ZS
+c4lShKK6BrPTJYs4NG1DGzmpToTnwoqZ8fAmi2XlZnuchC4NPSZaPATHvNIzt+z1
+PHo35D/f7j2pO1S8BCysQDHCbM5Mnomnq5aYcKCsdbh0czchOm8bkinLrYrKpii+
+Tk7pwL7TjRKLXkomm5D1Umds++pip8wH2cQpf93at3VDcOK4N7EwoIJB0kak6pSz
+Eu4I64U6gZs7tS/dGNSljf2OSSnRr7KWzq03zl8l75jy+hOds9TWSenLbjBQUGR9
+6cFr6lEUfAIEHVC1L68Y1GGxx4/eRI82ut83axHMViw1+sVpbPxg51Tbnio1lB93
+079WPFnYaOvfGAA0e0zcfF/M9gXr+korwQTh2Prqooq2bYNMvUoUKD85gnJ+t0sm
+rWrb8dee2CvYZXD5laGtaAxOfy/VKNmwuWuAh9kc
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+MIIGdjCCBN6gAwIBAgIRAITLsPr7KXlVB0GeWV9ucXQwDQYJKoZIhvcNAQEMBQAw
+VDELMAkGA1UEBhMCR0IxGDAWBgNVBAoTD1NlY3RpZ28gTGltaXRlZDErMCkGA1UE
+AxMiU2VjdGlnbyBQdWJsaWMgQ29kZSBTaWduaW5nIENBIFIzNjAeFw0yMjAxMjEw
+MDAwMDBaFw0yMzAxMjEyMzU5NTlaMHkxCzAJBgNVBAYTAlVTMRIwEAYDVQQIDAlN
+aW5uZXNvdGExKjAoBgNVBAoMIVZlcnRjb2luIFByb2plY3QgQ29kZSBTaWduaW5n
+IExMQzEqMCgGA1UEAwwhVmVydGNvaW4gUHJvamVjdCBDb2RlIFNpZ25pbmcgTExD
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEApYvYSFFnSWe9v1doi6xt
+5UXhsN8+70HbEZ7aBuhUGELS0RCcPj69kHra4rMjUDXMmmi6XlOUv9A2kZK26/Pj
+9XJjOog/T9zDJD+mJ8ysxt6wR5hfg0STtGNQYxeALoGdDOM4vifff/fpvdKLnW2T
+dRg0ZbW5+5MnL35n9B/Cl8h0tpPhrmaeK/hmfdk6wytnSEVDxHQ22OyBGqlujwkZ
+JyqdCiIVPMGLZ5SJKQ+mb6Kz7/3gDszG8rjVCjl8aQZblZ1rN1ekJxfMgvg7Q6cr
+tEuXSt0qInUaKO8+i6SO2hP7fm2PbhdtJOKvI2Ol1x/29keZoBZQ0h+WYXuLW8SQ
+ig6g7ZNhjyU0Gpqy1vK3tA11zW65sxstmnxHjN0PuaDAZk9BF5IspHU6mLVPuN+N
+kRwdFeLgzPFfjI1T0MITLWUi5DuIVJyGYRV9vSgneJeEJ4cLL3YyhDZjBGR0DLvl
+lgZVRI3EZyy7XKxbpBcbJFYkvE+XZ9hFgR6iqdufYaEK7Uw4FwDIIfgvPiS0L01D
+XIZgTFbQr5xUaN4DwBxOCNRbyhncHaSirq2jnMFCBWYCvncHecTrVpAY45YNdzC1
+SAcDZE66LF8DsGFpxQNGuNsaWUwH9DHTPEPLghFIQ50Al51y2tEmGS5LhJrD/uBb
+xK1jF9a3xM73VPTnYpVLeccCAwEAAaOCAZwwggGYMB8GA1UdIwQYMBaAFA8qyyCH
+KLjsb0iuK1SmKaoXpM0MMB0GA1UdDgQWBBSw530B1UHQXG7dlBgrNMNZg65YtzAO
+BgNVHQ8BAf8EBAMCB4AwDAYDVR0TAQH/BAIwADATBgNVHSUEDDAKBggrBgEFBQcD
+AzARBglghkgBhvhCAQEEBAMCBBAwSgYDVR0gBEMwQTA1BgwrBgEEAbIxAQIBAwIw
+JTAjBggrBgEFBQcCARYXaHR0cHM6Ly9zZWN0aWdvLmNvbS9DUFMwCAYGZ4EMAQQB
+MEkGA1UdHwRCMEAwPqA8oDqGOGh0dHA6Ly9jcmwuc2VjdGlnby5jb20vU2VjdGln
+b1B1YmxpY0NvZGVTaWduaW5nQ0FSMzYuY3JsMHkGCCsGAQUFBwEBBG0wazBEBggr
+BgEFBQcwAoY4aHR0cDovL2NydC5zZWN0aWdvLmNvbS9TZWN0aWdvUHVibGljQ29k
+ZVNpZ25pbmdDQVIzNi5jcnQwIwYIKwYBBQUHMAGGF2h0dHA6Ly9vY3NwLnNlY3Rp
+Z28uY29tMA0GCSqGSIb3DQEBDAUAA4IBgQAsxM5uF7WTyYveKq03lWRAP/DIdrNQ
+CZf9SUg78KoJva38dBCdrE8bTMKgJlIja14M5f77rsDdI5Mi0YxBjTLAAfe8H5dX
+Xgtutg6aQtbiObdNNodUVk73urdEutMlieN/dA+p/v4IK88JMtwWr6tmhFWklSm/
+nge/PeXDt1PkvlA7LOC1JX3wKoXtV88No0YtbmSYqhfGA6Vo6RqVRxC/k/mll7qt
+as3BIVQful56hxIYm23xtckNHGPQZ3A1d8ypHA3z2NDIINUK0RyhGACreg2VNogY
+nwNZnIJXy4WmtFctJtZ2jVCVyeC1IMNXKV6xOKPpL6lqP4CUF6jtLOgIR4Mcyde9
+ZglNKaUOBEfVyO7G8abyWFIG8U1e/+xoM56SO+hCOJuY9Cf3oojTV7w7jxaHn/xd
+FHxOU09fliveaTiE4fJvShu6yf56N9Tf5VZZDTf6yspR1j5hNsg18V9zvzslWixQ
+3+TO1McMFOJw4PxH2HOupRw86dqWaOS0TvU=
+-----END CERTIFICATE-----
+
```

Comment:
This cert is actually invalid. Let's leave this alone while we don't have our own cert.

Reply:
Changes to 'win-codesign.cert' have been backed out of this commit.